topics/variant-analysis/tutorials/mapping-by-sequencing/tutorial.md

```diff
@@ -0,0 +1,505 @@
+original late-flowering phenotype were recovered and their genomic DNA was
+pooled and subjected to whole-genome sequencing. 
+Together with whole-genome sequencing data of a pure L*er* background strain,
+this outcrossed pool data forms the starting point of our analysis.
+The L*er* strain sample, thus, can inform us about the markers that the
+outcrossed F2 pool *could* have inherited and exclusion of L*er* markers from
+a particular genomic region from the F2 pool provides evidence that the
+phenotype causing variant lies close to this region. With respect to
+[Figure 1](#figure-1), this means that we will only use "blue diamond" marker
+variants introduced through the mapping cross for the linkage analysis and
+ignore potentially informative (green and orange) variants found in the
+original mutant line.
+
+For your convenience, the sequenced reads from both datasets have already been
+mapped (using **Bowtie2**) to the *A. thaliana* Col reference genome (version
+TAIR10) so the two datasets take the form of BAM files with the mapping results
+(see the tutorial on
+[NGS data mapping](../../../sequence-analysis/tutorials/mapping/tutorial.html)
+if you do not know what this means).
+
+> ### {% icon hands_on %} Hands-on: Data upload and preprocessing
+>
+> > ### {% icon tip %} Tip
+> > This section assumes that you already know
+> >
+> > - how to upload data to Galaxy via links
+> > - edit dataset attributes
+> > - do some basic text manipulation on datasets
+> >
+> > An excellent introduction into these topics can be found in the
+> > [From peaks to genes](../../../introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html)
+> > tutorial (specifically, in the sections on
+> > [Data upload](../../../introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html#data-upload)
+> > and
+> > [File preparation](../../../introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html#file-preparation)).
+> {: .tip}
+> 
+> 1. Import the two BAM datasets representing the:
+>    - [L*er* mapping strain](https://zenodo.org/record/1098034/files/Ler_mapping_strain.bam)
+>    - [outcrossed F2 pool](https://zenodo.org/record/1098034/files/outcrossed_F2.bam)
+> 2. Specify the genome that was used at the reads mapping step
+>    1. Click on **Edit attributes** (the pencil icon displayed with the dataset)
+>    2. Select `Arabidopsis thaliana TAIR10` as the **Database/Build**
+>    3. **Save** the edited attributes
+> 3. Import the reference genome
+>
+>    > ### {% icon tip %} Tip
+>    > This step is only necessary if the TAIR10 version of the *A. thaliana*
+>    > genome is not available as a built-in genome on your Galaxy server.
+>    {: .tip}
+>
+>    1. Upload the reference genome using the URL:
+>       `https://www.arabidopsis.org/download_files/Genes/TAIR10_genome_release/TAIR10_chromosome_files/TAIR10_chr_all.fas`
+>    2. Truncate the title lines in the fasta dataset to just the bare
```

Comment:
A cosmetic comment...
We are trying to make the name of tool more obvious (like in the first few words) and also having first the name of the parameters and then the value. 
Do you think something we could change the next steps to something like?

```
>    2. Run **Replace parts of text** {% icon tool %} to truncate the title lines in the fasta dataset to just the bare chromosome names with the:
>       - "Find pattern" to `^>(\w+).*`
>            
>            This will find entire fasta title lines, or more specifically, any lines starting with `>` followed by at least one letter, digit or underscore)
>
>       - "Replace with" to `>chr$1`
>           
>            `$1` in it refers to the parenthesized part of the find pattern, *i.e.*, just the substring of letters/digits/underscores directly following the `>`
>
>       - "Find-Pattern is a regular expression" to `Yes`
```
It will render something like:

![screen shot 2018-01-23 at 16 56 42](https://user-images.githubusercontent.com/1842467/35285733-715c97a2-005e-11e8-8266-6579dbf18177.png)

What do you think? And do you think we could have something similar for the other hands-on?